### DISCUSSION TITLE
"Internal Error: Invalid type in descriptor" when using UM Tally

### DISCUSSION THREAD

**Discussion by kevinm387** [svalinn/DAGMC#843] [Q&A]
Hi DAGMC community,

I'm getting an error when running MCNP 6.2 with DAGMC 3.2 using an unstructured mesh tally. Here's a portion of the output:
```
Creating dagmc mesh tally54, input: mesh_in.h5m, output: mesh_out.h5m
    Tally range has psize: 1
  There are 630 tetrahedrons in this tally mesh.
  Tally mesh has 1423 triangles.  Building KD tree of size 2053... done.

Internal Error: Invalid type in descriptor
```

This error occurs when running on a server, but my local machine is able to run the same problem without errors. Does anyone have ideas on what is causing this error and what I could do to fix it? Thanks, I really appreciate the help!

**Comment by gonuke** [comment]
Hi @kevinm387 

Sorry it had taken so long to respond.

We'd need to see a little more info to try to understand the problem.  Can you share the lines from your input file that relate to this tally?

Paul

**Reply by kevinm387** [reply]
Hi Paul, thanks for the response! Here are some lines from another input that produced the same `Internal Error: Invalid type in descriptor` 
```
FC114 dagmc type=unstr_track inp=mesh_in.h5m
     out=mesh_out.h5m
FMESH114:N geom=dag
c 
c 
FC124 dagmc type=unstr_track inp=mesh_in.h5m
     out=mesh_out.h5m
FMESH124:P geom=dag
```
This simulation ran fine when these lines were commented out. It seems that DAGMC doesn't like my `type=unstr_track` selection, but I'm not sure how to fix that. Please let me know if there's any other useful information I could provide. Thanks again for your help!
